packages/core/admin/admin/src/pages/ProfilePage/components/Preferences/index.js

```diff
@@ -0,0 +1,171 @@
+import React from "react";
+import PropTypes from 'prop-types';
+import {
+  Typography,
+  Box,
+  Grid,
+  GridItem,
+  Flex,
+  Select,
+  Option
+} from '@strapi/design-system';
+import { useIntl } from 'react-intl';
+import upperFirst from 'lodash/upperFirst';
+
+const Preferences = ({ onChange, values, localeNames, allApplicationThemes }) => {
+  const { formatMessage } = useIntl();
+  const themesToDisplay = Object.keys(allApplicationThemes).filter(
+    (themeName) => allApplicationThemes[themeName]
+  );
```

Comment:
Isn't this just `Object.values`?

Reply:
Oh we're checking if `object[value]` has an actual value and remove if not remove the key from the array, yes of course.

Reply:
this looks like it's returning only the keys that have values, so not Object.values @joshuaellis